GPII-4218: Send UIO+ settings to Morphic

jobara opened the pull request:
https://issues.gpii.net/browse/GPII-4218

Requires changes to Universal for accepting the settings from UIO+ for Morphic.

jobara committed:
GPII-4218: Send UIO+ settings to Morphic

jobara commented:
@klown this is a draft PR containing the UIO+ related changes for [GPII-4218](https://issues.gpii.net/browse/GPII-4218)

amb26 commented:
Cheers - I assume this will be updated in the light of https://github.com/fluid-project/uio-plus/pull/17

jobara commented:
@amb26 thanks for merging that. I'll make a new dev release of UIO+ and update this PR with that. However, it won't really affect the code, as I've re-implemented the `updateSettings` invoker here because it needs to have a fallback to the default settings in the case that no settings are returned from Morphic.

jobara committed:
GPII-4218: Updating to latest dev release of UIO+

jobara committed:
GPII-4218: Add an event relay for changeSettingsReceived messages.

Currently we do not provide any special handling for these messags other 
than relaying into an event. However, this is a rough in implementation 
in case we require to wait for or in some other way respond to these 
confirmations.

jobara committed:
GPII-4218: Added a note about Morhpic Classic compatibility

jobara commented:
@amb26 given the state of Morphic Classic, should we tag and create GitHub release before merging this PR?

amb26 commented:
Yes, please issue a release and indicate in its release notes that it is the last release compatible with the last official release of Morphic Classic, 1.3.5

jobara commented:
@amb26 I've tagged and created a [GitHub Release](https://github.com/GPII/gpii-chrome-extension/releases/tag/v0.1.0-beta.14). I didn't bother publishing to the Chrome Web Store because that will get overwritten with what we publish after this PR is merged.

jobara committed:
GPII-4218: bumping version number

jobara commented:
@amb26 I bumped the version number to "0.2.0" as I figure this will be the last release and will drop the beta from tags and etc. If you think I should keep with the beta versioning I can switch it up.

see: [Manifest - Version](https://developer.chrome.com/apps/manifest/version)

jobara commented:
@amb26 when you review this PR, could you also update the description of the repository? I don't seem to have access to do that. Could you please change "UI Options Plus Chrome Extension" to be "UIO+ For Morphic"?
